src/main/kotlin/se/uulm/snowballr/backend/model/SnowballRException.kt

```diff
@@ -24,7 +24,17 @@ sealed class SnowballRException(
         entityType: EntityType,
         entityId: String,
         identifierType: IdentifierType = IdentifierType.ID,
-    ) : SnowballRException("${entityType.singularUpper()} with ${identifierType.displayName} '$entityId' not found.")
+    ) : SnowballRException("${entityType.singularUpper()} with ${identifierType.displayName} '$entityId' not found.") {
+        class ComposedId(
+            entityType: EntityType,
+            firstEntityId: String,
+            secondEntityId: String,
+            identifierType: IdentifierType = IdentifierType.ID,
+        ) : SnowballRException(
+            "${entityType.singularUpper()} with ${identifierType.displayName} '$firstEntityId' and " +
+                "'$secondEntityId' not found.",
+        )
+    }
```

Comment:
I see that this is unfortunate with two IDs. We could use varargs and a helper method to combine both and make it for N IDs.

The helper method could look like this:
```kotlin
fun displayEntityIds(entityIds: List<String>, identifierType: IdentifierType = IdentifierType.ID): String {
    var idString = entityIds.dropLast(1).joinToString(
        separator = ", ",
    ) { "'$it'" }

    if (entityIds.isNotEmpty()) {
        idString += " and '${entityIds.last()}'"
    }

    return "with ${identifierType.displayName} $idString"
}
```

Furthermore, we can remove the separate exception.
```suggestion
        vararg entityIds: String,
        identifierType: IdentifierType = IdentifierType.ID,
    ) : SnowballRException("${entityType.singularUpper()} ${displayEntityIds(entityIds.toList(), identifierType)} not found.")
```

There are some occurrences where `identifierType` is used. There we have to insert `identifierType = ` in front of the parameter value.

Reply:
This a great suggestion. I didn't think about generalizing this error and also not about the varargs in kotlin. But this makes it way better. Thank you.

Reply:
I have moved the helper function into a separate class, which can be extended if other errors also need helper functions. I think it makes sense to only have the error classes in the `SnowballRException` file. But if you think different let me hear your thoughts.